cusub_cortex/state_machine/config/mission_config.yaml

```diff
@@ -53,19 +53,28 @@ tasks:
       outcome_timed_out: 'skip'
     seconds_in_position: 2
     new_dv_check_rate: 30
-    mag_target: 152600 # target dvector magnitude
-    elev_target: 0.0
-    elev_thresh: 0.05 # how close elevation has to be
-    slay_carrot_dist: 1 # how far to go ahead to hit jiangshi
+    approach_carrot_dist: 2 # when approaching jiangshi, how far ahead to set drive setpoint
```

Comment:
I feel like at some point we should have a config file that is checked in specific to the sub, and all these changes for testing/ local sims should be saved locally and read in. Maybe like a bool that the code is on the sub vs. on a devel system. It would be annoying to have to go through configs on the sub and undo any development changes